homeassistant/components/pglab/discovery.py

```diff
@@ -0,0 +1,228 @@
+"""Discovery a PG LAB Electronics devices."""
+
+import json
+
+from pypglab.device import Device
+from pypglab.mqtt import Client
+
+from homeassistant.components.mqtt import ReceiveMessage
+from homeassistant.components.mqtt.subscription import (
+    EntitySubscription,
+    async_prepare_subscribe_topics,
+    async_subscribe_topics,
+    async_unsubscribe_topics,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr, entity_registry as er
+from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+from homeassistant.helpers.entity import Entity
+
+from .const import (
+    _LOGGER,
+    CREATE_NEW_ENTITY,
+    DEVICE_ALREADY_DISCOVERED,
+    DISCOVERY_TOPIC,
+    DOMAIN,
+)
+
+
+def GetDeviceIdFromDiscoveryTopic(topic: str) -> str | None:
```

Comment:
This is not OK naming of a function, rename it to `get_device_id_from_discovery_topic`, same comment for other functions.

Reply:
I'm going to change the function name as you suggested.
In the same file I'm going to rename  `CleanDiscoveredDevice` in `clean_discovered_device`